src/rust/engine/graph/src/entry.rs

```diff
@@ -62,10 +62,25 @@ pub(crate) enum EntryState<N: Node> {
   // The previous_result value is _not_ a valid value for this Entry: rather, it is preserved in
   // order to compute the generation value for this Node by comparing it to the new result the next
   // time the Node runs.
+  //
+  // A note on dirty as was_dirty:
```

Comment:
```suggestion
  // A note on dirty versus was_dirty:
```

Reply:
Replaced with different docs on the new enum